plugins/modules/monitoring/statsd.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: statsd
+short_description: Send metrics to StatsD
+description:
+  - The C(statsd) module sends metrics to StatsD.
+  - For more information, see U(https://statsd-metrics.readthedocs.io/en/latest/).
+  - Supported metric types are C(counter) and C(gauge).
+  - Currently unupported metric types are C(timer), C(set), and C(gaugedelta).
```

Comment:
I would put that into the same paragraph:
```suggestion
  - Supported metric types are C(counter) and C(gauge).
    Currently unupported metric types are C(timer), C(set), and C(gaugedelta).
```